database/migrations/2022_11_07_171403_add_config_descriptions.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+return new class() extends Migration {
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		DB::table('configs')->where('key', 'version')->update(['description' => 'Current version of Lychee']);
+		DB::table('configs')->where('key', 'check_for_updates')->update(['description' => 'Automatically check for new updates']);
+		DB::table('configs')->where('key', 'sorting_photos_col')->update(['description' => 'Default column used for sorting photos']);
+		DB::table('configs')->where('key', 'sorting_photos_order')->update(['description' => 'Default order used for sorting photos']);
+		DB::table('configs')->where('key', 'sorting_albums_col')->update(['description' => 'Default column used for sorting albums']);
+		DB::table('configs')->where('key', 'sorting_albums_order')->update(['description' => 'Default order used for sorting albums']);
+		DB::table('configs')->where('key', 'imagick')->update(['description' => 'Enable/Disable imagick processing']);
```

Comment:
```suggestion
		DB::table('configs')->where('key', 'imagick')->update(['description' => 'Enable imagick processing']);
```
Just to be consistent (we don't use `/Disable` anywhere else). Also, should it be `ImageMagick` perhaps?